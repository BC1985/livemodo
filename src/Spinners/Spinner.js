import React from "react";
import "./spinner.css";

function Spinner() {
  return <div className="spinner-3" />;
}

export default Spinner;
