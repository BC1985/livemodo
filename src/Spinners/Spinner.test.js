import React from "react";
import ReactDOM from "react-dom";
import Spinner from "../Spinners/Spinner";
import renderer from "react-test-renderer";

it("renders without crashing", () => {
  const div = document.createElement("div");
  ReactDOM.render(<Spinner />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("render UI as expected", () => {
  const tree = renderer.create(<Spinner />).toJSON();
  expect(tree).toMatchSnapshot();
});
