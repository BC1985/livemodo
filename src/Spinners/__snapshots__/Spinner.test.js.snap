// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render UI as expected 1`] = `
<div
  className="spinner-3"
/>
`;
